Auto suggest email address when logging in.

arjunvijayvargiya opened the pull request:
This feature will add some functionality to the log-in screen.It will suggest email addresses typed by the user earlier in his/her phone.So that user wont be typing it again.This will save some type from user perspective.
Apart from that it remembers the email id used by the user in his previous log-in.

updates Done
1)Removed extra object of Values class that was emailVal
2)removed old readnewprefs method and created new getters and setters for lastloginemailaddress
3)updated APP_PREFERENCE_NEWPREFS with new name of constant

gitanshu commented:
You don't have to create a new pull request every time you push a new commit. If you go back to your previous PR, you can see that your latest commit was added to that PR. Once you send a PR from your branch to other, all commits added to that branch get added to that PR automatically. 
However, a better practice would be to create a new pull request if your new commit is a corrective commit like this one. In such a case, you should merge all your commits and create a single new commit and then create a PR. This way, the commits with wrong variable names etc won't be merged into the main repo and thus, repo look cleaner.Look for 'git rebase' for squashing commits together.

arjunvijayvargiya commented:
Thanks for the advise @gitanshu .I will keep that in mind from the next time

vshivam commented on src/main/java/ly/priv/mobile/utils/Utilities.java:
```diff
@@ -43,6 +47,32 @@ public static boolean isValidEmail(String emailToCheck) {
+     * @param {String}  username that comes from last login
+     * @return {Boolean}
+     */
+    public static Set<String> emailIdSuggestor(Context context, String username) {
```
What if you remove the username as a parameter from this method and directly fetch it using `getLastLoginEmailAddress` inside this method itself. You wont need to do  `usernameextract = mValues.getLastLoginEmailAddress();` inside the LoginActivity. You do not need the last login email id anywhere else in the that activity. The usernameextract variable name also makes little sense, you could just call it lastLoginEmailAddress ?

vshivam commented:
@gitanshu That was a good suggestion :+1: 
@arjunvijayvargiya Just one more little change and the PR is good to go.

arjunvijayvargiya commented:
Hi shivam,
My pull requested is updated now .I have removed the parameter username.I have removed the unnecessary variable username extract too .please see to it once.
